packages/integrations/react/server.js

```diff
@@ -58,6 +59,12 @@ async function getNodeWritable() {
 }
 
 async function renderToStaticMarkup(Component, props, { default: children, ...slotted }, metadata) {
+	let prefix;
+	if (this && this.result) {
+		prefix = incrementId(getContext(this.result))
```

Comment:
nit: we can avoid exposing the `getContext` function and use it inside `incrementId` directly. `getContext` is only used in this place, so we don't need to export it.

Reply:
resolution: removed getContext